### PR TITLE
Not eligible contact us change

### PR DESCRIPTION
**Pull request by HarrisonShawAdvania**

### CODE REVIEW

File: CheckYourEligibility-Admin/Views/Check/Outcome/Not_Eligible_LA.cshtml [hunk 1]
```diff
@@ -22,10 +22,10 @@
     <p class="govuk-body"><a class="govuk-link" rel="noreferrer noopener" target="_blank" href="~/Home/Guidance">See a complete list of acceptable evidence (opens in new tab)</a>.</p>
 
     <h2 class="govuk-heading-m">If you have not received any evidence</h2>
-    <p class="govuk-body">You can either:</p>
+    <p class="govuk-body">You can either contact:</p>
     <ul class="govuk-list govuk-list--bullet">
-        <li>contact the relevant school to ask them to collect it from the parent or guardian</li>
-        <li>email the Department for Education support desk to request a separate check at ecs.admin@education.gov.uk</li>
+        <li>the relevant school to ask them to collect it from the parent or guardian</li>
+        <li>the <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-URYbNo1-R1EubHA0gNJis5UOVo3VU1KV0E0QkxTMFQ0RFZQUUZCR0E0WC4u">Department for Education support desk</a> to request a separate check</li>
```
Comment (marten-wetterberg) [inline CheckYourEligibility-Admin/Views/Check/Outcome/Not_Eligible_LA.cshtml:28]: If there's a way to not hardcode this value, even if just in appsettings.json I'd prefer that.

File: CheckYourEligibility-Admin/Controllers/CheckController.cs [hunk 2]
```diff
@@ -227,6 +228,19 @@ public async Task<IActionResult> Loader()
             }
         }
 
+        private string GetApplicationRegisteredText(CheckEligibilityStatus status)
```
Comment (marten-wetterberg) [inline CheckYourEligibility-Admin/Controllers/CheckController.cs:231]: Could we make this into a simple if statement inside the actual cshtml instead?

**Review by marten-wetterberg — CHANGES_REQUESTED** [review]

File: CheckYourEligibility-Admin/Views/Check/ApplicationsRegistered.cshtml [hunk 3]
```diff
@@ -32,25 +37,23 @@
...
             </tbody>
         </table>
     </div>
 </div>
+
 <div class="govuk-width-container">
     <div class="govuk-grid-row">
         <div class="govuk-grid-column-full">
             <h2 class="govuk-heading-m">What to do next</h2>
             <p> @TempData["Status"]</p>
```
Comment (marten-wetterberg) [inline CheckYourEligibility-Admin/Views/Check/ApplicationsRegistered.cshtml:53]: This is where the if clause could be. I believe we know what it is without having to use tempdata, just look at the status of the returned freshly minted application

**Review by marten-wetterberg — CHANGES_REQUESTED** [review]

File: CheckYourEligibility-Admin/Views/Check/Outcome/Not_Eligible_LA.cshtml (hunk 1 above)
Comment (HarrisonShawAdvania) [reply CheckYourEligibility-Admin/Views/Check/Outcome/Not_Eligible_LA.cshtml:28]: That is a good suggestion I'll look into that

**Pull request closed without merging**